### ISSUE TITLE
Staging Review finding: Missing headings

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#86830]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** VYE Team
**Product:** Verify Your Enrollment
**Feature:** Verify Your Enrollment

## Findings details
**VA.gov Experience Standard - issue:** [User can't determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/mgib-enrollments/benefits-profile/](https://staging.va.gov/education/verify-school-enrollment/mgib-enrollments/benefits-profile/), the "Pending documents" text is styled to look like an H2 heading but is coded as a `p` paragraph. Likewise, after submitting an enrollment verification, the success alert is styled as having a heading but the heading itself is just coded as a `div`.

Screen reader users often navigate the page via heading, in which they skip directly to the heading for the section of content they're looking for. When heading text isn't coded as a heading, it's not available for them as a navigational tool.

<details><summary>Link, screenshot or steps to recreate</summary>

The two examples identified are currently coded as:
```
<p class="vads-u-font-size--h2 vads-u-font-family--serif vads-u-font-weight--bold">Pending documents</p>
```
and
```
<div slot="headline" class=" vads-u-font-size--h2 vads-u-font-weight--bold vye-h2-style-as-h3 ">You have successfully verified your enrollment</div>
```

</details>

## Recommended action

Both of these should be H2 headings.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by wafimohamed** [comment]
done with this issue https://github.com/department-of-veterans-affairs/vets-website/pull/31687
